src/main/java/org/opentripplanner/routing/algorithm/filterchain/ItineraryListFilterChainBuilder.java

```diff
@@ -275,6 +279,11 @@ public ItineraryListFilterChainBuilder withRemoveTimeshiftedItinerariesWithSameR
     return this;
   }
 
+  public ItineraryListFilterChainBuilder withRideHailingFilter(ItineraryListFilter services) {
+    this.rideHailingFilter = services;
```

Comment:
Rename `services`

Reply:
Done in 84ceac77ac8788fa68bdc8b5d2951ae6a72edca0